iina/ko.lproj/FontPickerWindowController.strings

```diff
@@ -1,12 +1,15 @@
 /* Class = "NSTableColumn"; headerCell.title = "Family"; ObjectID = "2Ld-md-U2v"; */
 "2Ld-md-U2v.headerCell.title" = "서체";
 
-/* Class = "NSTextFieldCell"; placeholderString = "Type to filter…"; ObjectID = "DwS-ve-MLJ"; */
+/* Class = "NSTextFieldCell"; placeholderString = "Type to filter..."; ObjectID = "DwS-ve-MLJ"; */
```

Comment:
Why did this line change…… Not to blame you, just a little bit curious

Reply:
It might be my fault during search and replace three dots with ellipses.